src/test/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer has started work.");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped working.");
```

Comment:
Similarly, the stop message could be more informative. Consider changing it to something like: 'The Bulldozer has stopped working and is now idle.'